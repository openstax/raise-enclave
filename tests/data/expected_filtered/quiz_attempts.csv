id,assessment_id,user_uuid,course_id,attempt_number,grade_percentage,time_started,time_finished
6,1,d35a986b-a7ad-4c61-8e76-205eccaf9a06,3,1,0.0,1650122648,1661272078