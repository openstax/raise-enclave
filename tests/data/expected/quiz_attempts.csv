id,assessment_id,user_uuid,course_id,attempt_number,grade_percentage,time_started,time_finished
0,0,c6a4a01e-b112-475f-82ff-b051ab1d44e1,2,1,0.0,1650122648,1661272078
1,0,66730745-7ee0-444b-904e-c137c5334aab,2,1,0.0,1661273542,1661273642
2,1,c6a4a01e-b112-475f-82ff-b051ab1d44e1,2,1,40.0,1560012648,1561252186
3,1,c6a4a01e-b112-475f-82ff-b051ab1d44e1,2,2,80.0,1600112648,1661272186
4,1,66730745-7ee0-444b-904e-c137c5334aab,2,1,60.0,1661273564,1661273664
5,1,757555fb-3044-4164-99a9-5705d3bf4d5a,2,1,20.0,1661273250,1661273350
6,1,d35a986b-a7ad-4c61-8e76-205eccaf9a06,3,1,0.0,1650122648,1661272078